templates/webapi/main/index.html.ep

```diff
@@ -81,7 +81,7 @@
                 %>
             </div>
 
-            <div class="form-group">
+            <div class="form-group" id="filter-interval-content">
```

Comment:
When fixing things here anyways, could you also take care that the `input` elements contained by this `div` have unique IDs?

Reply:
Yes, you're right. Fixed